keep/api/tasks/process_event_task.py

```diff
@@ -4,7 +4,7 @@
 import json
```

Comment:
I would move anything related to incidents to another task, it complicates a lot